### DISCUSSION TITLE
rotating camera

### DISCUSSION THREAD

**Discussion by axe020** [openglobus/openglobus#743] [General]
I wanted to rotate the camera around the planet and also wanted to control the speed of it can someone please help me.
thank you

**Comment by Zemledelec** [comment]
Hey @axe020! 

Could you be more specific about what kind of rotation you want to see? 

Any camera has a position coordinates, you can use `set(eye, look, up) `methods in planet or renderer through _activeCamera_ pointer, for instance `planer.camera.set(new Vec3(100,100,100))`, also you can change eye property of the camera,` camera.eye.set(x,y,z)` and then `camera.update()`.  For controlling speed rotation you need to use `handler.deltaTime` it's available trough `planet.rendere.handler.deltaTime`, `deltaTime `- is time of the last frame, you can use this value for differentiation by time. for example distancePerFrame = speed * deltaTime, where speed - is your real speed. It's important for camera animation defined by real speed.

**Reply by axe020** [reply]
Hi @Zemledelec 
thanks for replying
I wanted to make globe spin around and when we interact with it spin animation should stop for a while
